### ISSUE TITLE
[Request]: A counter on the toolbar icon

### ISSUE THREAD

**Issue by tnilzon** [alexpdraper/reading-list#90]
[Request]: A counter always visible on the toolbar icon showing the number of stored items. If possible with an option to view both total number of items and also number of unread items (in different colors).

**Comment by alexpdraper** [comment]
I can put a badge the toolbar icon. Right now the badge shows a checkmark if the page you are on is on the Reading List. Would people rather see the total number of items on their list instead?

**Comment by tnilzon** [comment]
Yes (I can only speak for myself), but I would very much like a counter instead. Thanks.

**Comment by Marvinoo** [comment]
Yes please. A counter would be way more useful here.
